tests/Mock/Exception/MockErrorCodeException.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Mock\Exception;
+
+use ApiPlatform\Core\Exception\ErrorCodeExceptionInterface;
+
+class MockErrorCodeException extends \Exception implements ErrorCodeExceptionInterface
```

Comment:
The `Mock` part of the name is completely redundant. It's already in the namespace.

Reply:
removed